sound/soc/sof/sof-client-probes.c

```diff
@@ -34,19 +34,9 @@ struct sof_probes_priv {
 	struct snd_soc_card card;
 
 	const struct sof_probes_host_ops *host_ops;
+	const struct sof_probes_ipc_ops *ipc_ops;
 };
 
-struct sof_probe_point_desc {
-	unsigned int buffer_id;
-	unsigned int purpose;
-	unsigned int stream_tag;
-} __packed;
-
-struct sof_probe_dma {
-	unsigned int stream_tag;
-	unsigned int dma_buffer_size;
-} __packed;
-
 struct sof_ipc_probe_dma_add_params {
```

Comment:
Do you need to keep any of the ipc3 structures in the generic code?

Reply:
Only those that are needed in the probes ipc ops API. Possibly the API will be changed to use more generic structs, in which case the ipc3 structs can be local to ipc3 file.

Reply:
Thanks, and that is not the only struct I left lying around. Will also remove all ipc3 msg structs from sof-client-probes.c as they are not needed there anymore, as they are already in sof-client-probes-ipc3.c.